### ISSUE TITLE
Much wanted: create page with latest added episodes and latest added Movies ..

### ISSUE THREAD

**Issue by aroundmyroom** [danmed/KODIBrowser#4]
Much wanted: create page(s) with latest added episodes and latest added Movies ..

with info about date added etc.. 
not randomly ..

**Comment by danmed** [comment]
the default movies view and default tv view are both the latest added.. unless i'm misunderstanding

**Comment by aroundmyroom** [comment]
yes, I have seen the sql query, but it's not about the title of the tv series, but about what episodes of a series is being added. Call it 'Recently Added Episodes'
